README.md

```diff
@@ -11,14 +10,12 @@ forms.
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember Power Select v3 or above
-* Ember Bootstrap v3 or above
+* Ember Power Select v4 or above
+* Ember Bootstrap v4 or above
 * Ember.js v3.16 or above
 * Ember CLI v2.13 or above
```

Comment:
Should we increase Ember CLI 3.15+ as it's for Ember Bootstrap v4?

```suggestion
* Ember CLI v3.15 or above
```

Reply:
:+1: